clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,6 +30,7 @@ class GoogleConfigurationProperties {
     String accountType
     String project
     boolean alphaListed
+    boolean httpLoadBalancingEnabled // TODO(jacobkiefer): Feature flag for L7 development.
```

Comment:
Seems to me like you'd want a global flag to do this, not just per account. 

I just put it at the top level of the `google` yaml stack: https://github.com/spinnaker/clouddriver/commit/17bc7b24a605a4c7331313c0f61d5d07d5d9449f#diff-f3efbf7dc0f934c8fb5a295b1bf8e629R56